adminapp/src/pages/OfferingPickListPage.jsx

```diff
@@ -76,9 +100,10 @@ export default function OfferingPickListPage() {
     const pid = offeringProduct.product.id;
     productIdsAndQuantities[pid] ||= 0;
     productIdsAndQuantities[pid] += quantity;
-    fulfillmentIdsAndProductQuantities[fulfillmentOption.id] ||= {};
-    fulfillmentIdsAndProductQuantities[fulfillmentOption.id][pid] ||= 0;
-    fulfillmentIdsAndProductQuantities[fulfillmentOption.id][pid] += quantity;
+    // Represent no fulfillment with zero index
+    fulfillmentIdsAndProductQuantities[fulfillmentOption?.id || 0] ||= {};
```

Comment:
`const fullfillmentOptionId = fulfillmentOption?.id || 0`, no need to repeat this line 3 times.